### ISSUE TITLE
Estimated time to merge a PR is 13 minutes; is that the fastest we can do?

### ISSUE THREAD

**Issue by masih** [filecoin-project/go-f3#738]
Investigate how much we can make the merging process faster without affecting the benefits. 13m for a PR merge is too long.

**Comment by Stebalien** [comment]
I'm trying new runners:

1. 2xlarge takes 10m (down from 12m).
2. 4xlarge takes 7m.

I'm not really sure if that's worth it, but we can try it for a while. I'm starting with 4xlarge because 2xlarge definitely isn't worth it. And the machines _are_ spun up on demand so they shouldn't be too expensive...

**Comment by masih** [comment]
Yes too expensive for the time gain. Thank you for trying those out

**Comment by Stebalien** [comment]
I've dropped us back to the defaults for now. TBH, we likely just need to speed up the tests a bit.

**Comment by Kubuxu** [comment]
We could narrow down the test set in the merge queue. Most of our repos run zero tests in that phase.
Getting our tests would be nice as well but the above is lowest cost.
